Applications/DataExplorer/DataView/StationTreeModel.cpp

```diff
@@ -102,6 +102,17 @@ vtkPolyDataAlgorithm* StationTreeModel::vtkSource(const std::string &name) const
     return nullptr;
 }
 
+void StationTreeModel::setNameForItem(const std::string &stn_vec_name, std::size_t id, std::string item_name)
```

Comment:
Agree, but having it const qualified it assures that the `id` is not modified accidentally. The binary code is same.